tox.ini

```diff
@@ -19,3 +19,12 @@ deps =
     yamllint==1.15.0
 commands =
     bash -c "yamllint eve/main.yml salt/**/*.yaml"
+
+[testenv:tests]
+deps =
+    -r{toxinidir}/tests/requirements.txt
+passenv = 
+    SSH_CONFIG_FILE
+    TEST_HOSTS_LIST
+commands =
+    bash -c "pytest --ssh-config={env:SSH_CONFIG_FILE:<(VAGRANT_CWD={toxinidir} vagrant ssh-config)} --hosts={env:TEST_HOSTS_LIST:bootstrap} {posargs} tests"
```

Comment:
Not related to this line, but consider adding the following lines to this file to get rid of the `cryptography` deprecation warnings:

```
+
+[pytest]
+filterwarnings =
+    ignore:encode_point has been deprecated on EllipticCurvePublicNumbers and will be removed in a future version. Please use EllipticCurvePublicKey.public_bytes to obtain both compressed and uncompressed point encoding.:UserWarning
+    ignore:Support for unsafe construction of public numbers from encoded data will be removed in a future version. Please use EllipticCurvePublicKey.from_encoded_point:UserWarning
```

Reply:
Done